keyboardShortcuts.md

```diff
@@ -38,3 +38,13 @@
 * gco = git checkout
 * gst = git status
 * gcam = git commit -a -m (git commit amended files with message)
+* st (file name) = open up the file name with SublimeText
+* stt = open up the current directory in SublimeText 
```

Comment:
st and stt only work with a ZSH package installed, So they wont work for everyone. I would consider taking them out or making an addendum to state the zsh package to install :)